test/interface/test_filters.py

```diff
@@ -1151,3 +1165,36 @@ def test_complex_using_tuples(self, session):
 
         assert len(result) == 1
         assert result[0].id == 3
+
+
+class TestApplyArrayFilters:
+
+    @pytest.mark.usefixtures('multiple_corges_inserted')
+    def test_value_in_array(self, session, is_postgresql):
+        if not is_postgresql:
+            pytest.skip(ARRAY_NOT_SUPPORTED)
+
+        query = session.query(Corge)
+        filters = [{'field': 'tags', 'op': 'any', 'value': 'foo'}]
+
+        filtered_query = apply_filters(query, filters)
+        result = filtered_query.all()
+
+        assert len(result) == 1
+        assert result[0].id == 1
+        assert result[0].tags == ['foo']
+
+    @pytest.mark.usefixtures('multiple_corges_inserted')
+    def test_value_not_in_array(self, session, is_postgresql):
```

Comment:
We could rename this to something like `test_not_all_values_in_array` to reflect the operator's action.

Reply:
fixed with b0e1b4f